src/building/building_state.c

```diff
@@ -380,13 +386,18 @@ void building_state_load_from_buffer(buffer *buf, building *b, int building_buf_
     // Building state variables are automatically set to 0, so if the savegame version doesn't include
     // that information, you can be assured that the game will read it as 0
     
-    if (building_buf_size >= BUILDING_STATE_TOURISM_BUFFER_SIZE) {
+    if (building_buf_size >= BUILDING_STATE_CULTURE_BUFFER_SIZE) {
```

Comment:
This is not entirely correct. You must keep the old if as it was, then add a new if with `if (building_buf_size >= BUILDING_STATE_CULTURE_BUFFER_SIZE)`.

In the end it should look like this:

```
    if (building_buf_size >= BUILDING_STATE_TOURISM_BUFFER_SIZE) {
        b->house_arena_gladiator = buffer_read_u8(buf);
        b->house_arena_lion = buffer_read_u8(buf);
        b->is_tourism_venue = buffer_read_u8(buf);
        b->tourism_disabled = buffer_read_u8(buf);
        b->tourism_income = buffer_read_u8(buf);
        b->tourism_income_this_year = buffer_read_u8(buf);
    }
    if (building_buf_size >= BUILDING_STATE_CULTURE_BUFFER_SIZE) {
        b->house_theater_accessibility = buffer_read_u8(buf);
        b->house_amphitheater_accessibility = buffer_read_u8(buf);
        b->house_school_accessibility = buffer_read_u8(buf);
        b->house_library_accessibility = buffer_read_u8(buf);
        b->house_academy_accessibility = buffer_read_u8(buf);
    }
```